libs/sdk-ui-filters/src/AttributeFilter/Components/ElementsSelect/StatusBar/AttributeFilterShowFilteredElements.tsx

```diff
@@ -41,14 +53,12 @@ export const AttributeFilterShowFilteredElements: React.FC<IAttributeFilterShowF
                     alignPoints={ALIGN_POINTS}
                     arrowOffsets={ARROW_OFFSETS}
                 >
-                    <FormattedMessage
-                        id="attributesDropdown.relevantValues.tooltip"
-                        values={{
-                            child: attributeTitle,
-                            parents: parentFiltersTooltipText,
-                            strong: (chunks: ReactNode) => <strong>{chunks}</strong>,
-                        }}
-                    />
+                    {" "}
```

Comment:
this creates some intended space?

Reply:
This seems to be created by prettier. I removed it as it was not needed.